people/kwlai/kwlai.yaml

```
@@ -0,0 +1,5 @@
+- path: people/kwlai/bio-kwlai.qmd
```

```suggestion
- path: bio-kwlai.qmd
```

At least on my local computer, the hyperlink from your square in "Team" was ending up as `http://localhost:6256/people/kwlai/people/kwlai/bio-kwlai.qmd` (note the doubled `people/kwlai/`), which didn't work properly.

I think this edit will fix it, but if it wasn't a problem on your end, it may be a problem with my Rstudio configuration

---

When I remove `people/kwlai/` on my end and render, the link from my square shows "Not found". And I don't see a double `people/kwlai` in the hyperlink on my end.

---

@kristinawlai the plot thickens: I just set up online PR previews, and I'm seeing the same behavior there as on my local:  https://ucd-serg.github.io/pr-preview/pr-8/team.html

I think there might be something different about how your computer is compiling the website. It might be a quarto version issue; try running `quarto --version` in the Terminal? Github is using 1.6.40 and I've got 1.6.39.